Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM eclipse-temurin:latest
```

Comment:
The tag latest refers to an image with JDK 21. However the jar was build with JDK 17 in the workflow.

I suggest to use eclipse-temurin:17-jre for the image build.